backend/src/main/java/com/benine/backend/camera/ipcameracontrol/IPCamera.java

```diff
@@ -402,13 +405,13 @@ public String toJSON() throws CameraConnectionException {
     JSONObject json = new JSONObject();
     json.put("id", this.getId());
     try {
-      json.put("pan", getPosition().getPan());
-      json.put("tilt", getPosition().getTilt());
-      json.put("zoom", (double) getZoomPosition());
-      json.put("focus", (double) getFocusPosition());
-      json.put("autofocus", isAutoFocusOn());
-      json.put("iris", (double) getIrisPosition());
-      json.put("autoiris", isAutoIrisOn());
+      json.put("pan", new Double(getPosition().getPan()));
```

Comment:
Why is it required to create a new object from an already existing double object?